luminoth/predict.py

```diff
@@ -1,18 +1,26 @@
+import numpy as np
+
 import click
 import json
 import os
+import skvideo.io
```

Comment:
This needs ffmpeg installed right? Do it fail when importing or when trying to use it? Should we catch the error manually?

Reply:
It does fail, will catch the error